src/instructlab/system/info.py

```diff
@@ -130,6 +131,33 @@ def _instructlab_info():
     return {f"{name}.version": ver for name, ver in pkgs}
 
 
+def _default_paths() -> dict[str, str]:
+    """Default directories used by ilab"""
+    # d = {
```

Comment:
I kept this in the commit because the other idea I had was enumerating the defaults, but this seems way too messy and cluttered. 

Maybe there's a better place where these can go? `ilab config show` currently renders the *active* paths used by ilab, but not necessarily the defaults.